wpilibc/src/main/native/cpp/Joystick.cpp

```diff
@@ -16,15 +18,9 @@
 
 using namespace frc;
 
-const int Joystick::kDefaultXAxis;
-const int Joystick::kDefaultYAxis;
-const int Joystick::kDefaultZAxis;
-const int Joystick::kDefaultTwistAxis;
-const int Joystick::kDefaultThrottleAxis;
-const int Joystick::kDefaultTriggerButton;
-const int Joystick::kDefaultTopButton;
-static Joystick* joysticks[DriverStation::kJoystickPorts];
-static bool joySticksInitialized = false;
+constexpr int Joystick::kMinNumAxes;
```

Comment:
constexpr without initializer?

Reply:
Since it's static, it has to be declared here. It's initialized in the header. We do this elsewhere in wpilibc as well.